mambaSharedFramework/HLS Utils/Collections/HLSTagArray+RenditionGroups.swift

```diff
@@ -19,6 +19,16 @@
 
 import Foundation
 
+/// the hls file type (i.e. master vs media/variant vs indeterminate)
+public enum FileType {
```

Comment:
does it make sense to keep this public when the Playlists have their own classes now, not sure what a good public access case is for this now.

Reply:
Yeah, I think it should be public.

We still use this enum. It's returned by a utility function on `Array where Element == HLSTag`, and it's how the `Parser` makes a call on returning `VariantPlaylist` vs `MasterPlaylist`.

I'd like it to be public so that, if someone has some exotic needs and wants to make custom playlist types, they can still use this method to determine master vs variant if they need to without writing their own.